File: check_medical_data.js
const { Pool } = require('pg');

const pool = new Pool({
  connectionString: process.env.DATABASE_URL
});

async function checkMedicalData() {
  try {
    // Check medical_consent table
    const result = await pool.query('SELECT * FROM medical_consent');
    
    console.log('\n=== MEDICAL CONSENT DATA ===');
    console.log(`Total records: ${result.rows.length}\n`);
    
    if (result.rows.length === 0) {
      console.log('❌ No medical data found in database');
    } else {
      result.rows.forEach((row, index) => {
        console.log(`Record ${index + 1}:`);
        console.log(`  Driver ID: ${row.driver_id}`);
        console.log(`  Allergies: ${row.allergies || '(none)'}`);
        console.log(`  Medical Conditions: ${row.medical_conditions || '(none)'}`);
        console.log(`  Medication: ${row.medication || '(none)'}`);
        console.log(`  Doctor Phone: ${row.doctor_phone || '(none)'}`);
        console.log(`  Consent Signed: ${row.consent_signed || '(none)'}`);
        console.log(`  Consent Date: ${row.consent_date || '(none)'}`);
        console.log('');
      });
    }
    
    // Also check total driver count
    const drivers = await pool.query('SELECT COUNT(*) FROM drivers');
    console.log(`Total drivers in system: ${drivers.rows[0].count}`);
    
    await pool.end();
  } catch (err) {
    console.error('❌ Database error:', err.message);
    process.exit(1);
  }
}

checkMedicalData();

File: old-test-scripts/check_medical_data.py
import psycopg2
import os
from datetime import datetime

# Get database URL from environment
db_url = os.environ.get('DATABASE_URL')
if not db_url:
    print('❌ DATABASE_URL environment variable not set')
    exit(1)

try:
    # Connect to database
    conn = psycopg2.connect(db_url)
    cursor = conn.cursor()
    
    # Check medical_consent table
    cursor.execute('SELECT * FROM medical_consent')
    results = cursor.fetchall()
    
    print('\n=== MEDICAL CONSENT DATA ===')
    print(f'Total records: {len(results)}\n')
    
    if len(results) == 0:
        print('❌ No medical data found in database')
    else:
        # Get column names
        col_names = [desc[0] for desc in cursor.description]
        
        for i, row in enumerate(results, 1):
            print(f'Record {i}:')
            for col, val in zip(col_names, row):
                print(f'  {col}: {val or "(none)"}')
            print()
    
    # Check total driver count
    cursor.execute('SELECT COUNT(*) FROM drivers')
    driver_count = cursor.fetchone()[0]
    print(f'Total drivers in system: {driver_count}')
    
    cursor.close()
    conn.close()
    
except Exception as e:
    print(f'❌ Database error: {str(e)}')
    exit(1)
